File: db4o.cs/src/com/db4o/cs/internal/messages/MCommittedCallBackRegistry.java
/* Copyright (C) 2007 Versant Inc. http://www.db4o.com */
package com.db4o.cs.internal.messages;

import com.db4o.cs.internal.*;

public class MCommittedCallBackRegistry extends Msg implements MessageWithResponse {

	public Msg replyFromServer() {
		ServerMessageDispatcher dispatcher = serverMessageDispatcher();
		dispatcher.caresAboutCommitted(true);
		return Msg.OK;
	}

}

File: db4o.net/Db4objects.Db4o.CS/Db4objects.Db4o.CS/CS/Internal/Messages/MCommittedCallBackRegistry.cs
/* Copyright (C) 2004 - 2011  Versant Inc.  http://www.db4o.com */

using Db4objects.Db4o.CS.Internal;
using Db4objects.Db4o.CS.Internal.Messages;

namespace Db4objects.Db4o.CS.Internal.Messages
{
	public class MCommittedCallBackRegistry : Msg, IMessageWithResponse
	{
		public virtual Msg ReplyFromServer()
		{
			IServerMessageDispatcher dispatcher = ServerMessageDispatcher();
			dispatcher.CaresAboutCommitted(true);
			return Msg.Ok;
		}
	}
}
